src/TypeScript/Notifications/IUnsupportedNodeVersionNotificationService .cs

```diff
@@ -31,9 +32,36 @@ public interface IUnsupportedNodeVersionNotificationService
     [PartCreationPolicy(CreationPolicy.Shared)]
     internal class UnsupportedNodeVersionNotificationService : IUnsupportedNodeVersionNotificationService
     {
+        private readonly INotificationService notificationService;
+        private readonly INotification notification;
+
+        private const string NotificationId = "sonarlint.nodejs.min.version.not.found.10";
+
+        [ImportingConstructor]
+        public UnsupportedNodeVersionNotificationService(INotificationService notificationService, 
+            IDoNotShowAgainNotificationAction doNotShowAgainNotificationAction)
+        {
+            this.notificationService = notificationService;
+
+            notification = new Notification(
+                id: NotificationId,
+                message: Resources.NotificationUnsupportedNode,
+                actions: new INotificationAction[]
+                {
+                    new NotificationAction(Resources.NotificationShowMoreInfoAction, _ => ShowMoreInfo()),
+                    doNotShowAgainNotificationAction
+                }
+            );
+        }
+
         public void Show()
         {
-            // TODO
+            notificationService.ShowNotification(notification);
+        }
+
+        private void ShowMoreInfo()
+        {
+            // todo: output pane? wiki page?
```

Comment:
I'm not sure what we should do here.
Preferably it should go to a documentation page, but which link? our wiki? We've avoided linking directly to specific wiki pages.

Reply:
The spec says a [wiki page](https://docs.google.com/document/d/1EoG_dIMb83G1P16fgabtDHSofHh6DUNWmZlrrQ_nNJU/edit), but doesn't specify whether it should be the root page or a specific page.

We should open a discussion with ITOps about redirects for documentation (or ask @kevin-hinz whether the doc team are already working on this).

Reply:
We are planning for redirects, in part, with the Martech Team. At this time, the final destination URLs are not defined and no clear process is established.

There are 2 methods to use now:

1. According to [this demo](https://christianheilmann.com/2021/08/03/redirecting-github-pages/), it is possible to redirect .md pages which should include GitHub wikis. 
2. Use [this page of redirects](https://redirect.sonarsource.com/) that we own. It is likely that no one team or person controls this page; we just need to find out where it lives! In this option, we can create a new folder for SonarLint/, your hyperlink points to some key on that page, the key is redirected to the wiki (now), and to the docs.sonarlint (after migration).

The easy solution is Option 1: create a wiki, link to it, and hope the .md redirect workflow works later.
Option 2: takes a search to find/edit the redirects page, and a plan to make it work well. @peeter-piegaze-sonarsource should be involved here.

In both cases, the URL should be updated in the Product when we have the final destinations available and the content will live in a wiki now. We should meet to choose which option to follow.